docs/content/concepts/assets/asset-jobs.mdx

```diff
@@ -20,43 +20,43 @@ description: Asset jobs are the main unit for materializing and monitoring Softw
 
 ---
 
-## Relevant APIs
+## Creating asset jobs
 
-| Name                                   | Description                                               |
-| -------------------------------------- | --------------------------------------------------------- |
-| <PyObject object="define_asset_job" /> | A function for defining a job from a selection of assets. |
+Asset jobs materialize a fixed set of assets each time they run. We'll use the following assets to demonstrate how to create a few asset jobs:
```

Comment:
I think its more accurate to say asset jobs define a selection of assets that can be materialized, either manually or via automation.